src/models/reader.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (connection, DataTypes) => {
+  const schema = {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: { args: true, msg: "Please enter a name" },
+        notNull: { args: true, msg: "Please enter a name" },
+      },
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        isEmail: { args: true, msg: "use a valid email address" },
+        notEmpty: { args: true, msg: "Please enter an email" },
+        notNull: { args: true, msg: "Please enter an email" },
+      },
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: { args: [8, 16], msg: "password too short or long" },
```

Comment:
This is open to interpretation but I would have a password length that could be longer than 16 characters, purely because auto generated strong passwords are longer than this. Just my opinion